### DISCUSSION TITLE
Where is the self-regulation explained?

### DISCUSSION THREAD

**Discussion by 0verEngineer** [jmcollin78/versatile_thermostat#596] [Q&A] [answered]
There are 5 different self-regulation options and i would like to know how they work and how they differ from each other, can someone answer this and put it back into the README?

In this screenshot of the README there are links that should lead to the explanation of the self-regulation, however there is no explanation
![image](https://github.com/user-attachments/assets/949c0625-71f5-47f6-8ff5-40c330114622)

**Comment by jmcollin78** [comment, marked as answer]
You have to open the section to search in the documentation. Yes, this is not easy. I will remove the sections.
